packages/core/react/src/WalletProvider.tsx

```diff
@@ -160,13 +160,15 @@ export function WalletProvider({
     }, [adapter, setWalletName]);
     return (
         <WalletProviderBase
-            {...props}
```

Comment:
y tho? I did this so that any additions to WalletProvider would get caught as TS errors if they were forgotten in Base.

Reply:
Yeah, I could see that, but it makes things a bit more complicated for the sake of something that almost never happens. When do we add props to the WalletProvider, and how likely are we to forget to use them if we need them?